webpackjs/components/DocklessData/AllTimeButton.js

```diff
@@ -1,3 +1,24 @@
 import React from "react";
 
+const allTimeSelect = ( getDaysInMonth ) => {
+    const today = new Date();
+    const endMonthIndex = today.getMonth();
+    const endYear = today.getFullYear();
+    const lastDay = getDaysInMonth(`${endMonthIndex + 1}`, `${endYear}`);
+    const allTimeRange = `'2018-4-1' and '${endYear}-${endMonthIndex + 1}-${lastDay}T23:59:59.999'`
+    return allTimeRange;
+}
+
+const AllTimeButton = ({ getDaysInMonth, onClickAllTime, onClickDataViewSelect }) => (
+    <button
+        type="button"
+        className="btn btn-primary"
+        id="js-all-time-button"
+        value={allTimeSelect(getDaysInMonth)}
+        onClick={function(event) {onClickDataViewSelect("allTime"); onClickAllTime(event)}}
```

Comment:
It turns out I don't actually need to pass in the dataViewSelect function at all, since no menu options need to be displayed to the user when they click "All Time" (as opposed to the other buttons where the user gets a dropdown menu or calendar). I went ahead and removed that function, so now only one function needs to be invoked. I will take note of this for the future, though, when needing to invoke more than one function. Thanks!